webapp/channels/src/components/settings_sidebar/settings_sidebar.tsx

```diff
@@ -78,13 +47,13 @@ export default class SettingsSidebar extends React.PureComponent<Props> {
         (e.target as Element).closest('.settings-modal')?.classList.add('display--content');
     };
 
-    public handleKeyUp = (tab: Tab, e: React.KeyboardEvent) => {
+    public handleKeyDown = (tab: Tab, e: React.KeyboardEvent) => {
```

Comment:
This is one of the "bonus" fixes. We need to prevent the keydown event to stop the modal from scrolling when you switch tabs with the down arrow